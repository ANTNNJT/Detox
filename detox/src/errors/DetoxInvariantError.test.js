const DetoxInvariantError = require('./DetoxInvariantError');

describe('DetoxInvariantError', () => {
  it('should append a Report-Issue hint', () => {
    expect(() => { throw new DetoxInvariantError('Should do better') })
      .toThrowErrorMatchingSnapshot();
  });

  it('.from(message) should wrap the error message ', () => {
    expect(DetoxInvariantError.from('Should do better')).toMatchSnapshot();
  });
});
