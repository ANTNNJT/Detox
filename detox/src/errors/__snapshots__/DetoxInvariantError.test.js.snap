// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetoxInvariantError .from(message) should wrap the error message  1`] = `
"DetoxInvariantError: Should do better
Please report this issue on our GitHub tracker:
https://github.com/wix/Detox/issues"
`;

exports[`DetoxInvariantError should append a Report-Issue hint 1`] = `
"Should do better
Please report this issue on our GitHub tracker:
https://github.com/wix/Detox/issues"
`;
